apps/wallet-mobile/src/yoroi-wallets/utils/utils.ts

```diff
@@ -142,13 +142,13 @@ export const Quantities = {
     const parts = sanitized.split(decimalSeparator)
     const isDec = parts.length >= 2
 
```

Comment:
Branch all logic as earlier as possible to group up the logic, void index accessing.
```suggestion
    let fullDecValue = sanitized
    const isDec = parts.length >= 2
    if (isDec) {
        const [int, dec] = sanitized.split(decimalSeparator)
        fullDecValue = `${parts[0]}${decimalSeparator}${parts[1].slice(0, precision)}1`
        
    }

```

Reply:
Aren't you index accessing anyway? isDec already checks that parts indeed has length 2, so I don't see the harm in using the index

Reply:
yap, but is just unreadable, part[0] (what is part), `1` hacky one to keep decimals, you have to scan the code and make an effort to understand. 
`const [whole, decimal] = sanitzedInput` it's read once, and you got it.

Reply:
@stackchain @jorbuedo that's now updated, and also added comment above the `1` to explain the use-case